openslides_backend/locale/translator.py

```diff
@@ -0,0 +1,30 @@
+import gettext
+from typing import Callable, Optional
+
+DEFAULT_LANGUAGE = "en_US"
+PATH_TO_LOCALE = "/app/openslides_backend/locale"
+DOMAIN = "messages"
+
+
+class TranslatorSingleton:
+    def __init__(self) -> None:
+        self.language = DEFAULT_LANGUAGE
+
+    def get_translate_function(self) -> Callable[[str], str]:
+        t = gettext.translation(DOMAIN, PATH_TO_LOCALE, [self.language])
+        if not t:
+
+            def null_translate(msg: str) -> str:
+                return msg
+
+            return null_translate
+        return t.gettext
+
+    def set_translation_language(self, lang: Optional[str]) -> None:
+        if lang is None:
+            self.language = DEFAULT_LANGUAGE
+        else:
+            self.language = lang
+
+
+Translator = TranslatorSingleton()
```

Comment:
I would like a function `translate` which accepts a string to be translated, fetches the translate function from the instance and returns the translated string. Then, you can import that via `... import translate as _` in other files and do not have to call the translator every time.

Reply:
I don't know how to do that.

Reply:
_(): Seems to have a special meaning. I can do from ....locale.translator import Translator and self.PW... = Translator(..) but not import Translator as _ and _(...).